packages/hello-gsm/src/Stores/StoreContainer.ts

```diff
@@ -8,12 +8,15 @@ interface storeType {
   mypageModalContent: string;
   isSearching: boolean;
 
+  showSideBar: boolean | null;
   setShowFAQModal: () => void;
   setFAQModalTitle: (title: string) => void;
   setFAQModalContent: (content: string) => void;
   setShowMypageModal: () => void;
   setMypageModalContent: (content: string) => void;
   setIsSearching: (trueOrFalse: boolean) => void;
+
+  setShowSideBar: (value: boolean | null) => void;
```

Comment:
```suggestion
  isSearching: boolean;
  showSideBar: boolean | null;
  
  setShowFAQModal: () => void;
  setFAQModalTitle: (title: string) => void;
  setFAQModalContent: (content: string) => void;
  setShowMypageModal: () => void;
  setMypageModalContent: (content: string) => void;
  setIsSearching: (trueOrFalse: boolean) => void;
  setShowSideBar: (value: boolean | null) => void;
```

이런식으로 코드를 관심사에 맞게끔 배치를 해보는게 어떨까용?

Reply:
좋아요 👍